### PR TITLE
Deployment fix

### LINKED ISSUE #90: Email notification
We achieved to get the user the email but it doesn't contain the link. Need to be solved
photo_2017-07-18_10-21-11

### PR DESCRIPTION
**Pull request by xeon-zolt**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message 
Fixes #90  
this is to fix deployment error and deleted another file not used `build.sh` and incremented to latest release ISO

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by xeon-zolt** [comment]
@meets2tarun please confirm that I have used the latest release link of ISO

File: build1.sh [hunk 1]
```diff
@@ -2,13 +2,14 @@
 #packages required to edit
 sudo apt-get install -qq squashfs-tools genisoimage
 #downloading the ISO to edit
-wget -q https://github.com/fossasia/meilix/releases/download/untagged-c9f68d2fa6b3d1cdbd99/meilix-zesty-20170611-i386.iso
+wget -q https://github.com/fossasia/meilix/releases/download/untagged-15efc7e1086f3686d999/meilix-zesty-20170718-i386.iso
```
Comment (meets2tarun) [inline build1.sh]: This is the latest one, but it should get varied automatically as other latest release come.

Comment (xeon-zolt) [inline build1.sh]: if i add fetch latest release option it can also fetch latest release made by user using meilix generator too

**Comment by meets2tarun** [comment]
Please paste a link to the release iso so that I test it

**Comment by xeon-zolt** [comment]
@meets2tarun you can receive it by email

_Sent from my YU5510 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request merged**